### ISSUE TITLE
AUC calculation

### ISSUE THREAD

**Issue by nmsommer** [RobRomijnders/LSTM_tsc#12]
Hi!
Thanks for your code.  In addition to accuracy, I am trying to obtain AUC values.  I am suspecting that I am doing something wrong in that accuracy results are good but AUC are below 0.5. 

This is what I have in the tsc_model.py file:
      a = tf.cast(tf.argmax(logits, 1),tf.float32) #AUC
      b = tf.cast(self.labels,tf.float32) #AUC
      auc = tf.contrib.metrics.streaming_auc(a,b) #AUC
      self.auc_value = tf.reduce_mean(tf.cast(auc, "float")) #AUC**

Then, in the main program, I have:
sess.run(tf.initialize_local_variables()) #for AUC
auc_test = sess.run([model.auc_value],feed_dict = {model.input: Xtest_batch, model.labels: ytest_batch, model.keep_prob:1.0})
print('Result of testing: AUC is %5.3f'%(auc_test))

***Please let me know if you spot an incorrect approach.  Thanks!
